### PR TITLE
development instructions for running all examples/ notebooks

### PR DESCRIPTION
**Pull request by jwmueller**
should not be merged until after: 
https://github.com/cleanlab/examples/pull/4 
is merged

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -63,7 +63,20 @@ which you can open with your web browser.
...
 You can check that the [examples](https://github.com/cleanlab/examples) still
-work with changes you make to cleanlab by manually testing them.
+work with changes you make to cleanlab by manually running the notebooks.
+You can also run all example notebooks as follows:
+
+```console
+git clone https://github.com/cleanlab/examples.git
+```
+
+Then specify your local version of cleanlab source in the first line of: **examples/requirements.txt**
```
Comment (anishathalye) [inline DEVELOPMENT.md:73]: Can you clarify what this means and give an example? E.g. "specify your local version of cleanlab as a relative path in ..., e.g. `../cleanlab` if `cleanlab` and `examples` are sibling directories.

**Comment by anishathalye** [comment]
Seems a little weird to require editing `requirements.txt`, but this seems fine for now and we can improve it later.

File: DEVELOPMENT.md (hunk 1 above)
Comment (jwmueller) [reply DEVELOPMENT.md]: done

**Comment by jwmueller** [comment]
> Seems a little weird to require editing `requirements.txt`, but this seems fine for now and we can improve it later.

Agreed.
 @JohnsonKuan feel free to edit your examples-PR to add way for user to instead easily specify local version of cleanlab or pypi version.

**Pull request merged**